cypress/integration/tests/developer/controls/businessservices/create.test.ts

```diff
@@ -63,7 +64,7 @@ describe("Business service validations", { tags: "@tier2" }, () => {
 
     it("Business service field validations", function () {
         // Navigate to business service tab and click "Create New" button
-        selectUserPerspective("Developer");
+        selectUserPerspective(migration);
         clickByText(navMenu, controls);
         clickByText(navTab, businessServices);
```

Comment:
Can we do BusinessServices.openList(); here ?

Reply:
Thanks for noticing this :)